packages/legend-graph/package.json

```diff
@@ -22,12 +22,16 @@
   "sideEffects": false,
   "type": "module",
   "exports": {
-    ".": "./lib/index.js",
+    ".": {
```

Comment:
I think we should follow the same strategy we have discussed before. Why can't we import anything you need from this module and expose that in `legend-vscode-extension-dependencies`? Why do you need to directly import `@finos/legend-graph`, also if you changed it this way, have you checked if this has any internal impact when we chain the projects together?

Reply:
This is little bit complicated. Better to explain over screen share. 
The graph compilation code exists  legend studio which takes array of presets. From legend-area, I want to create a cjs bundle of legend-area specific presets. (that bundle contains legend-graph).

The legend-studio cis bundle also contains legend-graph. Both cjs bundles contain legend-graph.  Legend graph has function guranteeType() which checks `instanceof` . Since legend-graph exists in both cis bundles, guranteeType fails. 

So, I have to keep a separate cis bundle of legend-graph which will be common/external to both cjs bundles of legend-area & finos/legend-studio. 

I have tested this approach and it works. I beleive this extra bundling is only required for legend-graph package for compiling the graph.